src/app-nordigen/services/nordigen-service.js

```diff
@@ -208,6 +209,7 @@ export const nordigenService = {
     );
 
     return {
+      iban: accountMetadata.iban,
```

Comment:
I did it like this because I figured that having services return data with processing it would be cleaner though that ideal sort of falls apart with `normalizeAccount` call anyway. Also, I also hought that having access to unhashed IBANs returned by services within the server app could perhaps be useful. Since having services return unhashed IBAN doesn't make one's data vulnerable as long as they're hashed before transmitting them over the network, it seemed fine.

But anyway, I've updated the service to hash IBANs before returning. I've also figured that instead of duplicating the logic that is bound to be identical in all custom bank integrations, it's better to just put that common logic into the service and as such, `hash` is now generated there instead based on `iban` field returned from `normalizeAccount`.

Reply:
Sorry to go back-and-forth with this, but after seeing [this](https://github.com/actualbudget/actual-server/pull/166/files#diff-d22b96a7983dbb8c5863145ede4630c7de55759d408ef77fcf231eb73ed63436L11) change I realize that your previous implementation was actually better (hashing only before returning the response).

We should keep the `mask` field as-is in order to be compatible with the Plaid interface and the current usage of bank-sync in Actual frontend.

Sorry about this.. 😓

Reply:
@MatissJanis Do you want me to revert the whole commit or just put the `mask` back into bank classes? And if you want the latter, should I generate the hash in the service (and therefore return unhashed iban in the bank classes) or should I returned hashed iban in bank classes?

Reply:
So `mask` should have the last 4 digits of the IBAN (not the hash; same as before).

Basically revert back to this commit: https://github.com/actualbudget/actual-server/pull/166/commits/668ea641e42556a22bcd3602b679a95cee724ecd

Again: sorry for misleading you with [this comment](https://github.com/actualbudget/actual-server/pull/166#discussion_r1145917093)

Reply:
> (not the hash; same as before).

Yeah, I know. The question about the hashed iban hash was about the `iban` field.

> Again: sorry for misleading you with [this comment](https://github.com/actualbudget/actual-server/pull/166#discussion_r1145917093)

No worries, it happens. I just pushed a revert commit so unless there's anything else, this should be good to go :)